newdle/client/src/TimelineExample.js

```diff
@@ -1,10 +1,21 @@
 import React from 'react';
-import Timeline from './components/Timeline';
 import moment from 'moment';
+import {useSelector} from 'react-redux';
+import {getCalendarActiveDate, getCalendarDates} from './selectors';
+import {serializeDate} from './util/date';
+import Timeline from './components/Timeline';
 
 export default function TimelineExample({participants}) {
   const availability = generateParticipantAvailability(participants);
-  return <Timeline date="2019-07-29" availability={availability} />;
+  const dates = useSelector(getCalendarDates);
+  const activeDate = useSelector(getCalendarActiveDate);
+
+  return (
+    <Timeline
+      date={activeDate ? serializeDate(activeDate) : dates[0]}
```

Comment:
What if `dates` is empty? Right now we have dummy dates, but once timeslots are in Redux I think we'll only have the dates where timeslots exist, so we should probably default to today or tomorrow there.

Reply:
Also, why is this still called `TimelineExample`? :D

Reply:
Ohh, right. I was thinking that we would always have some dates there. Anyway I will default to today and ofc will change the name :P